streams/src/test/java/org/apache/kafka/streams/kstream/RepartitionTopicNamingTest.java

```diff
@@ -522,182 +521,181 @@ public void process(final String key, final String value) {
-            "    Processor: joined-stream-other-windowed (stores: [joined-stream-other-join-store])\n" +
-            "      --> joined-stream-other-join\n" +
-            "      <-- KTABLE-TOSTREAM-0000000011\n" +
-            "    Processor: joined-stream-this-windowed (stores: [joined-stream-this-join-store])\n" +
-            "      --> joined-stream-this-join\n" +
-            "      <-- KSTREAM-FILTER-0000000029\n" +
-            "    Processor: KSTREAM-AGGREGATE-0000000014 (stores: [KSTREAM-AGGREGATE-STATE-STORE-0000000013])\n" +
-            "      --> KTABLE-TOSTREAM-0000000018\n" +
-            "      <-- KSTREAM-SOURCE-0000000041\n" +
-            "    Processor: KSTREAM-REDUCE-0000000023 (stores: [KSTREAM-REDUCE-STATE-STORE-0000000022])\n" +
-            "      --> KTABLE-TOSTREAM-0000000027\n" +
-            "      <-- KSTREAM-PEEK-0000000021\n" +
-            "    Processor: joined-stream-other-join (stores: [joined-stream-this-join-store])\n" +
-            "      --> joined-stream-merge\n" +
-            "      <-- joined-stream-other-windowed\n" +
-            "    Processor: joined-stream-this-join (stores: [joined-stream-other-join-store])\n" +
-            "      --> joined-stream-merge\n" +
-            "      <-- joined-stream-this-windowed\n" +
-            "    Processor: KTABLE-TOSTREAM-0000000018 (stores: [])\n" +
-            "      --> KSTREAM-SINK-0000000019\n" +
-            "      <-- KSTREAM-AGGREGATE-0000000014\n" +
-            "    Processor: KTABLE-TOSTREAM-0000000027 (stores: [])\n" +
-            "      --> KSTREAM-SINK-0000000028\n" +
-            "      <-- KSTREAM-REDUCE-0000000023\n" +
-            "    Processor: joined-stream-merge (stores: [])\n" +
-            "      --> KSTREAM-SINK-0000000038\n" +
-            "      <-- joined-stream-this-join, joined-stream-other-join\n" +
-            "    Sink: KSTREAM-SINK-0000000012 (topic: outputTopic_0)\n" +
-            "      <-- KTABLE-TOSTREAM-0000000011\n" +
-            "    Sink: KSTREAM-SINK-0000000019 (topic: outputTopic_1)\n" +
-            "      <-- KTABLE-TOSTREAM-0000000018\n" +
-            "    Sink: KSTREAM-SINK-0000000028 (topic: outputTopic_2)\n" +
-            "      <-- KTABLE-TOSTREAM-0000000027\n" +
-            "    Sink: KSTREAM-SINK-0000000038 (topic: outputTopicForJoin)\n" +
-            "      <-- joined-stream-merge\n\n";
+        "   Sub-topology: 0\n" +
+        "    Source: KSTREAM-SOURCE-0000000000 (topics: [input])\n" +
+        "      --> KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-MAP-0000000001 (stores: [])\n" +
+        "      --> KSTREAM-FILTER-0000000002, count-stream-repartition-filter\n" +
+        "      <-- KSTREAM-SOURCE-0000000000\n" +
+        "    Processor: KSTREAM-FILTER-0000000002 (stores: [])\n" +
+        "      --> KSTREAM-MAPVALUES-0000000003\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-MAPVALUES-0000000003 (stores: [])\n" +
```

Comment:
Order is changing due to sorting the processors when generating the topology description.  The order of the processors in the physical plan is the same.